vignettes/calc.Rmd

```diff
@@ -262,7 +258,7 @@ set.seed(98121)
 mydt[, mywghts := sample(50:1300, 2000, replace = T)]
 
 # survey set the data
-mysvy <- srvyr::as_survey_design(mydt, weights = mywghts)
+mysvy <-dtsurvey::dtsurvey(data.table(mydt[, `_id` := NULL]), weight = 'mywghts')
```

Comment:
Somewhere we'll have to explain the syntax for telling dtsurvey if it  is admin data, a simple survey, or a survey with replicate weights. I'm guessing in a separate vignette?